recipes/iris-esmf-regrid/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "iris-esmf-regrid" %}
+{% set version = "0.1.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SciTools-incubator/iris-esmf-regrid/archive/v{{ version }}.tar.gz
+  sha256: 0d8d4b4dd73f32dd5d8f4789838e0ca76919373f216115266274bc5481fcaf0f
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux or py<3.6]
+
+requirements:
+  build:
+    - python
```

Comment:
@jamesp don't think you need a `build` section here

Reply:
Removed from the recipe 👍